src/inversion_scripts/invert_calcJ.py

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import glob
+import numpy as np
+import xarray as xr
+import yaml
+from scipy.interpolate import interp2d
+from netCDF4 import Dataset
+from src.inversion_scripts.utils import load_obj
+
+
+def do_inversion(
```

Comment:
This file seems to be nearly the same as `invert.py`, but with the end goal of calculating Ja. I dont think we need this file. My lognormal-errors branch updates invert.py to prion out Ja. @megan-he can you confirm whether we need this or not?

Reply:
I'm not seeing this file called from anywhere, so I agree that it should be removed. If it is needed in the future, perhaps @megan-he can submit a separate pull request.

Reply:
Now removed in https://github.com/geoschem/integrated_methane_inversion/pull/157/commits/c53f4d519860a678a5dec31f403770c497cfab43.

Reply:
Right this was just a temporary script, thanks!